src/BDCS/Projects.hs

```diff
@@ -26,6 +27,13 @@ import           Database.Esqueleto
 
 import BDCS.DB
 
+allProjects :: MonadIO m => SqlPersistT m [Projects]
+allProjects = do
+    vals <- select $ from $ \project -> do
```

Comment:
Looks good, but I think it should just be named projects.  We already have a groups and files.  You might want to consider making a projectsC, too.